snuba/migrations/snuba_migrations/sessions/0001_sessions.py

```diff
@@ -11,7 +10,8 @@
 from snuba.clusters.storage_sets import StorageSetKey
 from snuba.migrations import migration, operations, table_engines
 from snuba.migrations.columns import MigrationModifiers as Modifiers
-from snuba.processor import MAX_UINT32, NIL_UUID
+from .matview import aggregate_columns_v1, create_matview_v1
```

Comment:
you were commenting on this in one of the previous PRs. I moved the code to a common module since I want to dedupe it with the `0003` backward statement.
If this is not desired, I can revert this and rather copy-paste the first matview into the rollback statement.